web/src/components/BetaBar.test.tsx

```diff
@@ -13,8 +13,8 @@ describe("<BetaBar />", () => {
 
   it("opens feedback modal on button click", () => {
     render(<BetaBar />);
-    expect(screen.queryByText(Config.betaBar.betaModalTitle)).not.toBeInTheDocument();
+    expect(screen.queryByText(Config.feedbackModal.betaModalTitle)).not.toBeInTheDocument();
     fireEvent.click(screen.getByText(Config.betaBar.betaModalButtonText));
-    expect(screen.getByText(Config.betaBar.betaModalTitle)).toBeInTheDocument();
+    expect(screen.getByText(Config.feedbackModal.betaModalTitle)).toBeInTheDocument();
```

Comment:
I like this renaming. Would it make sense to extend to the fields also (ie, `feedbackModalTitle` instead of `betaModalTitle`)?